core/src/com/mygdx/game/client/screen/ChooseUnitFieldDialog.java

```diff
@@ -0,0 +1,58 @@
+package com.mygdx.game.client.screen;
+
+import com.badlogic.ashley.core.ComponentMapper;
+import com.badlogic.ashley.core.Entity;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Dialog;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.mygdx.game.client.component.FieldComponent;
+import com.mygdx.game.client.component.UnitComponent;
+import lombok.extern.java.Log;
+
+import java.util.logging.Level;
+
+@Log
+public class ChooseUnitFieldDialog {
```

Comment:
Wydaje mi się, że to powinno zostać napisane bez takiego polegania na static.

Podoba mi się, że to taki "DialogManager", który zapewnia, że jest tylko jeden widoczny dialog tego typu. 
Wydaje mi się, że później będzie takie podejście można rozszerzyć na resztę UI btw.